src/pages/roadmap.md

```diff
@@ -6,42 +6,43 @@ hide_table_of_contents: true
 
 Timelines are estimates and subject to change. For information about delivered features and fixes, see the [Changelog](/docs/changelog).
 
-## Release cycle
+## 2023 milestones
 
-We typically follow this release cycle:
+### Q4
 
-- Alpha: Super early peek into the feature being built. Not ready for use in production apps.
-- Beta: Opportunity to test, validate, and provide feedback. Not ready for use in production apps.
-- Production: Ready for use in production apps.
+- Allow/denylist user preferences available in all SDKs
 
-## Features roadmap
+## 2024 milestones
```

Comment:
- Each bullet below is based on a milestone on the public Milestones board: https://github.com/xmtp/roadmap/milestones?direction=asc&sort=due_date&state=open

- Instead of using MVP, can we use the established Alpha or Beta release cycle names? For now, anything that is an MVP on the milestone board, I have designated it as an alpha here.

- Each bullet links to the milestone issue listing all of the epics in the milestone. I think this is a good level of granularity for people to learn more about the work to be done. Instead of manually listing epic info (which could be quite volatile) on .org - we can link people to the epics themselves, which will naturally be accurate and up to date as the work progresses.

Reply:
These changes sound great, @jhaaaa - no objections here; thanks for proposing!